pkg/translate/translate_test.go

```diff
@@ -221,8 +221,7 @@ func TestNewTranslator(t *testing.T) {
 		{
 			name:         "version 1.5",
```

Comment:
please add one more test for 1.6, to make sure it will fallback to 1.5 instead of error.

Reply:
done